### PR TITLE
 Fix further schema issues

### PR DESCRIPTION
**Pull request by awilkins**
Fix a few more errors in the schemas ; database migrates fully from empty schema.

### CODE REVIEW

File: database/schema/V1.0__schema.sql [hunk 1]
```diff
@@ -1,126 +1,1356 @@
...
-    constraint reference_organisation_odscode_ck_1 check (char_length(trim(ods_code)) > 0),
-    constraint reference_organisation_odscode_ck_2 check (upper(ods_code) = ods_code),
-    constraint reference_organisation_orgstatus_ck check (char_length(trim(org_status)) > 0),
-    constraint reference_organisation_orgrecordclass_ck check (char_length(trim(org_record_class)) > 0),
-    constraint reference_organisation_postcode_ck check (char_length(trim(postcode)) > 0),
-    constraint reference_organisation_primaryroleid_ck check (char_length(trim(primary_role_id)) > 0),
-    constraint reference_organisation_primary_roledescription_ck check (char_length(trim(primary_role_description)) > 0),
-    constraint reference_organisation_organisationlink_ck check (char_length(trim(organisation_link)) > 0),
-    constraint reference_organisation_organisationname_ck check (char_length(trim(organisation_name)) > 0)
+--
```
Comment (ChristopherJamesMorris) [inline database/schema/V1.0__schema.sql]: Can we remove system generated comments.

File: database/schema/V1.0__schema.sql [hunk 2]
```diff
@@ -1,126 +1,1356 @@
...
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 14.2 (Debian 14.2-1.pgdg110+1)
+-- Dumped by pg_dump version 14.2 (Debian 14.2-1.pgdg110+1)
+
+SET statement_timeout = 0;
```
Comment (ChristopherJamesMorris) [inline database/schema/V1.0__schema.sql]: Are these SET commands actually needed/used? They were generated by the pg_dump utility?

Comment (awilkins) [reply database/schema/V1.0__schema.sql]: Seems to run OK without them

File: database/schema/V1.0__schema.sql (hunk 1 above)
Comment (awilkins) [reply database/schema/V1.0__schema.sql]: Done

File: database/schema/V1.0__schema.sql [hunk 3]
```diff
@@ -1,126 +1,1356 @@
...
+
+COPY application.user_session (user_session_id, user_id, start_time, end_time) FROM stdin;
+\.
+
+
+--
+-- Data for Name: lookup; Type: TABLE DATA; Schema: reference; Owner: postgres
+--
+
+COPY reference.lookup (lookup_id, lookup_value, lookup_type_id, added_date, disabled_date, linked_lookup_id) FROM stdin;
```
Comment (ChristopherJamesMorris) [inline database/schema/V1.0__schema.sql:382]: Do we need to make these actual INSERT statements? Could the hardcoded dates be replaced with now() ?

**Pull request merged**